### PR TITLE
Show single-value dropDowns as text in view mode

### LINKED ISSUE #3411: Show single-value dropdowns as text in view mode
### Is your feature request related to a problem? Please describe.

Properties with none or one value are rendered as disabled dropdown in view mode

image

### Describe the solution you'd like

Show those values as simple text

image

### Describe alternatives you've considered

_No response_

### Additional context

@jancborchardt @marcoambrosini @nimishavijay what do you think?

Ref https://github.com/nextcloud/contacts/issues/3284

### PR DESCRIPTION
**Pull request by hamza221**
closes #3411

### CODE REVIEW

File: src/components/Properties/PropertyGroups.vue [hunk 1]
```diff
@@ -56,6 +57,9 @@
 						{{ t('contacts', 'No results') }}
 					</span>
 				</NcSelect>
+				<p v-else>
+					{{ localValue.length ===0?'none': localValue.toString() }}
```
Comment (ChristophWurst) [inline src/components/Properties/PropertyGroups.vue:61]: l10n

File: src/components/Properties/PropertyGroups.vue [hunk 2]
```diff
@@ -56,6 +57,9 @@
 						{{ t('contacts', 'No results') }}
 					</span>
 				</NcSelect>
+				<p v-else>
+					{{ localValue.length ===0?t('contacts','none'): localValue.toString() }}
```
Comment (ChristophWurst) [inline src/components/Properties/PropertyGroups.vue:61]:
```suggestion
					{{ localValue.length === 0 ? t('contacts','none') : localValue.toString() }}
```

File: src/components/Properties/PropertyGroups.vue [hunk 3]
```diff
@@ -109,6 +113,9 @@ export default {
 	},
 
 	computed: {
+		showAsText() {
+			return this.isReadOnly && this.localValue.length <= 1
```
Comment (ChristophWurst) [inline src/components/Properties/PropertyGroups.vue:117]:
```suggestion
			return this.isReadOnly && this.localValue.length === 0
```

nitpick: easier to comprehend this way IMO

**Review by ChristophWurst — APPROVED** [review]
Works :sunglasses:

File: src/components/Properties/PropertyGroups.vue (hunk 3 above)
Comment (hamza221) [reply src/components/Properties/PropertyGroups.vue:117]: It supposed to be true for single-values too

Comment (ChristophWurst) [reply src/components/Properties/PropertyGroups.vue:117]: Sorry, you are right. I misread as <1 but it's <=1

**Pull request merged**